### PR TITLE
Fix make run on windows via msys2

### LINKED ISSUE #484: "make run" not working on Windows
Self-explanatory.  Ran through the Cowboy "getting started" tutorial on Windows using Msys2.  The code works, but "make run" does not appear to.

Copy/paste of "make run" on my system:

```
Vultaire@core MSYS ~/hello_erlang
$ make run
make[1]: Entering directory '/home/Vultaire/hello_erlang/deps/cowboy'
make[2]: Entering directory '/home/Vultaire/hello_erlang/deps/cowlib'
make[2]: Leaving directory '/home/Vultaire/hello_erlang/deps/cowlib'
make[2]: Entering directory '/home/Vultaire/hello_erlang/deps/ranch'
make[2]: Leaving directory '/home/Vultaire/hello_erlang/deps/ranch'
make[1]: Leaving directory '/home/Vultaire/hello_erlang/deps/cowboy'
===> Starting relx build process ...
===> Resolving OTP Applications from directories:
          c:/msys64/home/Vultaire/hello_erlang/ebin
          c:/msys64/home/Vultaire/hello_erlang/deps
          c:/Program Files/erl6.2/lib
          C
          \msys64\home\Vultaire\hello_erlang\apps;C
          \msys64\home\Vultaire\hello_erlang\deps
          c:/msys64/home/Vultaire/hello_erlang/_rel
===> Missing beam file elf_format <<"c:/Program Files/erl6.2/lib/hipe-3.11.1/ebin/elf_format.beam">>
===> Resolved hello_erlang_release-1
===> Including Erts from c:/Program Files/erl6.2
===> release successfully created!

Crash dump was written to: erl_crash.dump
init terminating in do_boot ()

Crash dump was written to: erl_crash.dump
init terminating in do_boot ()
/bin/sh: _rel/{"init: No such file or directory
erlang.mk:6241: recipe for target 'run' failed
make: *** [run] Error 127
```

References:
- https://twitter.com/Vultaire/status/685125093257658370
- http://pastebin.ca/3321003

### PR DESCRIPTION
**Pull request by headcrabmeat**
Fixes #484

### CODE REVIEW

File: plugins/relx.mk [hunk 1]
```diff
@@ -11,6 +11,7 @@ RELX_CONFIG ?= $(CURDIR)/relx.config
 RELX_URL ?= https://github.com/erlware/relx/releases/download/v3.19.0/relx
 RELX_OPTS ?=
 RELX_OUTPUT_DIR ?= _rel
+RELX_REL_EXT ?= ""
```
Comment (essen) [inline plugins/relx.mk]: The "" is unnecessary, otherwise looks fine.

Comment (headcrabmeat) [inline plugins/relx.mk]: good point

**Comment by essen** [comment]
I can't seem to make it work, but I don't think the problem is on your end. I get some Erlang crash dumps twice when trying to do `make`, even without your patch. No time to investigate right now unfortunately.

**Comment by essen** [comment]
I figured it out and will send a patch soon. However your PR doesn't work with it, can you take a look? I'll push in a few minutes.

**Comment by essen** [comment]
The behavior I'm seeing is that the release is started, but I don't have a console, FYI.

**Comment by headcrabmeat** [comment]
Sure you have to be in a "windowed" environment as it starts `werl`. I also see crash dumps but not sure the reason.

**Comment by essen** [comment]
Thanks, I'll retry in a windowed environment then. :-)

**Comment by headcrabmeat** [comment]
rebased

**Comment by headcrabmeat** [comment]
Did you have a chance to try it?

**Comment by essen** [comment]
Will do now.

**Comment by essen** [comment]
Yes it works, I will merge in a minute.

**Comment by essen** [comment]
Merged locally, push soon. Thanks!

**Comment by headcrabmeat** [comment]
Cool! Thank you.

**Pull request closed without merging**